README.md

```diff
@@ -1,8 +1,10 @@
 # Sal [![npm version](https://badge.fury.io/js/sal.js.svg)](https://www.npmjs.com/package/sal.js) [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/mciastek/sal/blob/master/LICENSE) [![Build Status](https://travis-ci.org/mciastek/sal.svg?branch=master)](https://travis-ci.org/mciastek/sal)
 
-Performance focused, lightweight (less than **2.5 kb**) scroll animation library, written in vanilla JavaScript. No dependencies!
+Performance focused, lightweight (less than **2.5 kb** without Gzip) scroll animation library, written in vanilla JavaScript. No dependencies!
```

Comment:
It's with Gzip. I think it should be 2.8kb. See screenshot below (gziped compression is on):
<img width="541" alt="screen shot 2018-09-11 at 19 47 15" src="https://user-images.githubusercontent.com/3169629/45377708-9cfc4f80-b5fb-11e8-89e6-8fa95561bd60.png">

Reply:
Oh, right, I looked only at the JS file and forgot about counting CSS as well!